src/features/advanced-content-insights/flowTypes.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+export type AdvancedContentInsightsType = {
+    /** Whether the advanced content insights FF is enabled */
+    enabled: boolean,
+    isActive: boolean,
+    onChange?: (isEnabled: boolean) => void,
+    ownerEId?: number,
+    userId?: number,
```

Comment:
@marcoartaviaq what are `ownerEId` and `userId` used for?

Reply:
Sorry, I forgot to answer this. These two props are sent from a parent app that would be used on a children's app (box content preview). On that children's app, we use them if present.

Reply:
oh so these props are only being used for Preview? if another app was going to use Advanced Content Insights, would they use these props as well?

does USM use it in the config? https://github.com/box/box-ui-elements/blob/d8315b512e89b7bc4539d93a6f5c988712b789ba/src/features/unified-share-modal/flowTypes.js#L323

we might want to rework how we're passing some of the props instead of putting them in one object. since `enabled` is only used as a feature flip, then it would eventually be removed right? that means that `isActive` would ideally be the only thing prop needed to toggle Advanced Content Insights?